docs/argo-server-sso.md

```diff
@@ -149,6 +149,7 @@ What this might look like in your chart configuration:
          - --auth-mode=sso
        sso:
          issuer: https://argo-cd.mydomain.com/api/dex
+         expiry: 240
```

Comment:
1. Could we add a similar note about the ability to determine the length of your login somewhere in this doc?

1. Could we add a quick inline comment here as to what this does, and its units (hours)